src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
```

Comment:
Consider using a more descriptive name for the new object, like 'invertedRobot' instead of 'obj'.